### DISCUSSION TITLE
how to extract whisper encodings

### DISCUSSION THREAD

**Discussion by nkumar248** [openai/whisper#2219] [Q&A]
Hey, I wanted to ask what the best way to extract whisper encodings is. I have tried using a fork by @IsaacOnline but it is giving me confusing results (have previously posted a question on this) so I'll try posting an open question.

**Comment by montesclarosglennbenedict** [comment]
You could try using the [SpeechBrain](https://github.com/openai/whisper/discussions/1246) library on it. I forget the implementation but it offers an alternative to the Isaac's fork.

In general, you want to process it based on [OpenAI's Cookbook] (https://cookbook.openai.com/examples/whisper_processing_guide), where you
1. Trim the audio to the relevant parts
2. Add relevant terminology to the encodings
3. Fix any unicode issues with the encodings
